pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -166,6 +169,10 @@ class BatteryModelOptions(pybamm.FuzzyDict):
                 Which electrode(s) intercalates and which is counter. If "both"
                 (default), the model is a standard battery. Otherwise can be "negative"
                 or "positive" to indicate a half-cell model.
+            * "x-average side reactions": str
+                Whether to average the side reactions (SEI growth, lithium plating and
+                the respective porosity change) over the x-axis in Single Particle
+                Models, can be "false" (default) or "true".
```

Comment:
shouldn't it be true for SPM? I think it won't make a difference to results, but will be faster

Reply:
I think I have fixed it now, take a look. It is a bit messy at the moment, hopefully will be clearer once we have derived the SPMe for cracks and loss of active material submodels.